src/app/client/src/components/PieChart.tsx

```diff
@@ -13,32 +13,41 @@ import {
   ChartTooltipContent,
   ChartConfig,
 } from "@/components/ui/chart";
+import { FeederMessage } from "../../../types/client-types";
 
-// Define the type for the pie chart prop
-interface PieChartComponentProps {
-  data: string[];
-}
-
-// PieChartComponent implementation (subcomponent)
-const PieChartComponent = ({ data }: PieChartComponentProps) => {
-  // Process the array of strings and count the occurrences
-  const processedData = useMemo(() => {
-    const counts: { [key: string]: number } = {};
+// Declare the classMap with prediction values and colors
+const classMap = {
```

Comment:
does this have to be defined in two places?

Reply:
Will fix tomorrow